acceptance/static_creds_test.go

```diff
@@ -0,0 +1,175 @@
+//go:build acceptance
+// +build acceptance
+
+package acceptance
+
+import (
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/roles"
+	"github.com/gophercloud/gophercloud/openstack/identity/v3/users"
+	"github.com/opentelekomcloud/vault-plugin-secrets-openstack/openstack"
+	"github.com/opentelekomcloud/vault-plugin-secrets-openstack/openstack/fixtures"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+)
+
+type testStaticCase struct {
+	Cloud      string
+	ProjectID  string
+	DomainID   string
+	Root       bool
+	SecretType string
+	Username   string
+	Extensions map[string]interface{}
```

Comment:
Optional: No need to make fields exported

Reply:
Here I export only fields that will be needed for the testcase, to specify different cases later. What is the better way?

Reply:
You don't need to export them to use inside package: https://go.dev/play/p/fMPsWJfI4Xq

Reply:
Changed fields to lowercase